public/index.html

```diff
@@ -29,6 +30,10 @@
   <body style="margin: 0; background-color: #1b1b1b;">
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <div class = "preload">
+      <img src="./visualrithmPreloader.gif" alt="robot processing">
+      <h2 id = "preloadText" >Please wait while we get things ready...</h2>
+    </div>
```

Comment:
move these lines inside <div id="root"> {Your code} </div>

Reply:
sorry, I'm unsure where it should be placed

Reply:
Ok move all these lines inside the div with id `root`